build-support/bin/check_banned_imports.py

```diff
@@ -9,6 +9,8 @@
 
 from common import die
 
+from pants.util.strutil import bullet_list, softwrap
```

Comment:
This means that you cannot run the script directly anymore. I don't think it's worth it